docs/source/guides/libraries/java.rst

```diff
@@ -86,12 +86,12 @@ to the root folder of your project:
 
   cp dist/libirohajava.jnilib /path_to_your_project
 
-If you use **Gradle**, you need to copy ``example/java/dist/iroha_lib.jar`` to the
+If you use **Gradle**, you need to copy ``example/java/dist/iroha-bindings-example.jar`` to the
```

Comment:
I think you need to copy both `iroha-bindings-example.jar` and `libirohajava.jnilib`. And 'libirohajava.jnilib` in linux has extension `libirohajava.so`.

Reply:
Also please rename the lib on line 82.

Reply:
and `.dll` for windows i guess